index.html

```diff
@@ -1854,6 +1854,53 @@ <h2>Default Language</h2>
           (e.g., en, de-AT, gsw-CH, zh-Hans, zh-Hant-HK, sl-nedis).</span>
         </span>
     </section>
+    <section id="sec-identifiers">
+      <h2>Identifiers</h2>
+      <p>
+        Web resources are identified by uniform resource identifiers (URIs).
+        The Thing Description defines the field <code>id</code> for that purpose, which is in the format "anyURI".
+        In deployments of multiple devices in the same application context, things must be distinguished by the consumer.
+        In these cases, these identifiers must be <em>unique</em>.
+        Use cases with worldwide deployments typically require <em>globally</em> unique identifiers.
+      </p> 
+      <section id="sec-identifiers-format">
+        <h3>Format</h3>
+      <p>
+        A common and well defined format for these identifiers enables easy implementation 
+        and simplifies handling across manufacturers through the use of libraries. 
+        It also ensures that there are no name clashes of devices from different manufacturers.
+        The [[UUID]] format has been developed to satisfy these requirements.
+      </p>
+        <p>
+          <span class="rfc2119-assertion" id="common-constraints-identifiers-uuid">
+            It is RECOMMENDED to use Universally Unique IDentifier (UUID) [[RFC4122]] as common format for identifers.
+          </span>
+        </p>
+          <span class="rfc2119-assertion" id="common-constraints-identifiers-random">
+          It is RECOMMENDED to use [[UUIDv4]] as common format for unique identifers, where random identifiers are required.
+          </span>
+      </p>
+      </section>
+      <section id="sec-identifiers-mutability">
+        <h3>Unique Identifiers</h3>
```

Comment:
Should this title be "Mutable Identifiers" or "Immutable Identifiers"?

Reply:
Both are valid titles, I prefer "Mutable identifiers"